Stripe/PublicHeaders/STPFormEncodable.h

```diff
@@ -0,0 +1,16 @@
+//
+//  STPFormEncodable.h
+//  Stripe
+//
+//  Created by Jack Flintermann on 10/14/15.
+//  Copyright © 2015 Stripe, Inc. All rights reserved.
+//
+
+#import <Foundation/Foundation.h>
+
+@protocol STPFormEncodable <NSObject>
+
+- (NSString *)rootObjectName;
+- (NSDictionary *)propertyNamesToFormFieldNamesMapping;
```

Comment:
(no need to change) 
I've taken to just calling dictionaries `keyToValue`, e.g. `propertyNameToFormFieldName`